GIS.md

```diff
@@ -0,0 +1,198 @@
+# Why Gnome-Initial-Setup
+There are several use cases where it makes sense for system installation and user creation to not happen at the same 
+time. In these instances, it is useful to have a way to skip the user creation step during installation and for it to 
+be handled later, perhaps even by a different user.
+
+To accommodate this workflow Ubuntu ships with Gnome Initial Setup, a program that can safely handle user account 
+creation and configuration. Gnome Initial Setup is started by default when a system is booted and GDM is unable to
+detect any user accounts. Instead of taking you to the login screen, a special Gnome Initial Setup user session is started.
```

Comment:
Do we want / need to call attention to the fact that this integration with GDM is something that the user will need to sort out themselves if they are using a different display manager?

Reply:
Good point, yes, let me make that explicit, took me awhile to figure it out myself :slightly_smiling_face: